src/main/java/io/crate/testing/CrateTestCluster.java

```diff
@@ -67,26 +67,36 @@ public class CrateTestCluster extends ExternalResource {
     private final String crateVersion;
     private final Map<String, Object> commandLineArguments;
 
+    private final int transportPortsFrom;
+    private final int transportPortsTo;
+
+    private final int httpPortsFrom;
+    private final int httpPortsTo;
+
+    private final int psqlPortsFrom;
+    private final int psqlPortsTo;
```

Comment:
Looks like these variables are never used (read)?

Reply:
hmm I will add some unit tests to check